en/get-started.md

```diff
@@ -125,28 +121,22 @@ minikube start
 You should see output like this, with some differences depending on your OS and hypervisor:
 
 ```
-😄  minikube v1.10.1 on Darwin 10.15.4
-✨  Automatically selected the hyperkit driver. Other choices: docker, vmwarefusion
-💾  Downloading driver docker-machine-driver-hyperkit:
-    > docker-machine-driver-hyperkit.sha256: 65 B / 65 B [---] 100.00% ? p/s 0s
-    > docker-machine-driver-hyperkit: 10.90 MiB / 10.90 MiB  100.00% 1.76 MiB p
-🔑  The 'hyperkit' driver requires elevated permissions. The following commands will be executed:
-
-    $ sudo chown root:wheel /Users/user/.minikube/bin/docker-machine-driver-hyperkit
-    $ sudo chmod u+s /Users/user/.minikube/bin/docker-machine-driver-hyperkit
-
-
-💿  Downloading VM boot image ...
-    > minikube-v1.10.0.iso.sha256: 65 B / 65 B [-------------] 100.00% ? p/s 0s
-    > minikube-v1.10.0.iso: 174.99 MiB / 174.99 MiB [] 100.00% 6.63 MiB p/s 27s
+😄  minikube v1.24.0 on Darwin 12.1
```

Comment:
Should we make it clear in L106 for the minimum version to use?

Reply:
Good question.  I can suggest users to use the newer versions of minikube like v1.24.0 but have no idea which minikube version is minimum requirement. I mean users can run TiDB on minikube using default configuration.

Reply:
OK, let's just suggest users to use the newer versions, thanks!

Reply:
Added another commit to suggest users to use the newer versions.